Source/Core/DolphinWX/NetPlay/NetWindow.h

```diff
@@ -105,7 +105,6 @@ class NetPlayDialog : public wxFrame, public NetPlayUI
 	void OnTraversalError(int error) override;
 
 	static NetPlayDialog*& GetInstance() { return npd; }
-	static NetPlayClient*& GetNetPlayClient() { return netplay_client; }
```

Comment:
this function returns a reference to a global variable, which doesn't make any sense. maybe it would be ideal to make the variable not global, but the easiest non-breaking change is to remove the function and use the global variable directly.